pkg/ccl/backupccl/alter_backup_schedule.go

```diff
@@ -115,6 +115,12 @@ func doAlterBackupSchedules(
 		return err
 	}
 
+	if s.fullJob == nil {
+		// This can happen if a user calls DROP SCHEDULE on the full schedule.
+		// TODO(benbardin): Can we prevent this state from occurring in the first place?
```

Comment:
Can we link https://github.com/cockroachdb/cockroach/issues/87435 as the desired state in this TODO. I think this would prevent this state from showing up.

Reply:
Yeah, that's a good idea!